docs/native-modules.md

```diff
@@ -218,6 +218,8 @@ namespace NativeModuleSample
 
 The `REACT_MODULE` macro-attribute says that the class is a React Native native module. It receives the class name as a first parameter. All other macro-attributes also receive their target as a first parameter. `REACT_MODULE` has an optional parameter for the module name visible to JavaScript and optionally the name of a registered event emitter. By default, the name visible to JavaScript is the same as the class name, and the default event emitter is `RCTDeviceEventEmitter`.
 
+Methods annotated with `REACT_METHOD` and friends must have the `noexcept` specifier, otherwise the program will not compile. Module authors should make sure all exceptions are handled inside the method.

```

Comment:
Can you make sure this addtion is added to the older doc files?

I.e. under `website/versioned_docs/version-x.xx/nativemodules.md`